js/src/forum/components/LinkPreview.js

```diff
@@ -42,7 +49,7 @@ export default class LinkPreview extends Component {
   }
 
   getLink(text) {
-    return m('a', this.linkAttributes, text);
+    return <Link {...this.linkAttributes}>{this.loading ? this.getDomain() : text ?? this.getDomain()}</Link>;
```

Comment:
There's a problem here that I didn't notice before, the `Link` component cannot be redirected to the destination page.

Reply:
Could you please elaborate on that using an example? I tested the Component and found no issue on my dev installation.